app/javascript/layouts/AssistanceLayout.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import Layout from "./Layout"
+import { PageHeader, SidebarBlock, t } from "@bloom-housing/ui-components"
+import { renderInlineMarkup } from "../util/languageUtil"
+import { ConfigContext } from "../lib/ConfigContext"
+
+import "./AssistanceLayout.scss"
+
+export interface Props {
+  children: React.ReactNode
+  title: string
+  subtitle: string
+}
+
+const AssistanceLayout = ({ children, title, subtitle }: Props) => {
+  const { getAssetPath } = React.useContext(ConfigContext)
+
+  return (
+    <Layout title={title}>
+      <PageHeader
+        title={title}
+        subtitle={subtitle}
+        inverse
+        backgroundImage={getAssetPath("bg@1200.jpg")}
+      />
+      <article className="flex flex-wrap flex-col md:flex-row relative max-w-5xl m-auto w-full">
+        <div className="w-full md:w-2/3" data-test-id="assistance-main-content">
+          {children}
+        </div>
+        <SidebarBlock
+          className="w-full md:w-1/3 text-gray-950 md:border-l border-t md:border-t-0 border-gray-450 p-6 mx-0"
+          title={t("footer.contact")}
+        >
+          <span className="text-gray-950">
+            <p className="mb-4">
+              <span className="font-semibold">
+                {t("assistance.contact.questionsAboutListings.title1")}
+              </span>
+              <br /> {renderInlineMarkup(t("assistance.contact.questionsAboutListings.subtitle"))}
+            </p>
+            <p className="mb-4">
+              <span className="font-semibold">{t("assistance.contact.helpLine.title1")}</span>
+              <br /> {renderInlineMarkup(t("assistance.contact.helpLine.subtitle"))}
+            </p>
+            <div>
+              <span className="font-semibold">
+                {t("assistance.contact.questionsAboutPrograms.title1")}
+              </span>
+              <br />
+              <span>
+                {renderInlineMarkup(t("assistance.contact.questionsAboutPrograms.contactWebsite"))}
+              </span>
+              <span className="mt-1">
+                {renderInlineMarkup(t("assistance.contact.questionsAboutPrograms.contactEmail"))}
+              </span>
+              <span className="mt-1">
+                {t("assistance.contact.questionsAboutPrograms.contactCall.title")}
+                <ul className="list-disc list-inside mt-1">
+                  <li>English: (415) 701-5622</li>
+                  <li>Españo: (415) 701-5623</li>
```

Comment:
few comments here cc @yindipei @emilyshore 

- is this missing an l? i.e. `Español`

- do we want each of these links to be translated for each language. For example, when displaying English on the Spanish page do we want it to display `Inglés`?

- consider adding tel links similar to comment above

Reply:
- Yes, it it should absolutely be `Español` 
- It should display similarly to the language bar at the top. So it remains the same no matter what language page the user is on. 
- I wondering if we need to add a tel link for it to show up on mobile phones. For some sites, the phone OS/browser automatically detects phone numbers. Not certain if this is true for 311 though. Will need to test the component on a mobile phone.

Reply:
- Totally right!
- I went back and forth on this, but I thought since we don't translate the language bar in the top then this was the preferred way? Would love @yindipei and @emilyshore's opinions on this!
- Definitely